.travis.yml

```diff
@@ -28,9 +28,23 @@ before_script:
 
 script:
   - make test-travis
+  - make
+  - make release
+
+deploy:
+  provider: releases
+  api_key: $GITHUB_TOKEN
+  file: 
+    - install-utask.sh
+    - utask
```

Comment:
Why are you uploading the binary?
Goreleaser already do that.

Reply:
my bad, wasn't aware it was redundant